packages/renderer/src/lib/appearance/IconImage.spec.ts

```diff
@@ -78,6 +79,7 @@ test('Expect valid source and alt text with light mode', async () => {
 
   // wait for image to be loaded
   await new Promise(resolve => setTimeout(resolve, 200));
+  await tick();
```

Comment:
having this both really feel like a bad practice, and maybe we should use `vi.waitFor` instead